camPod/Classes/CacheHelper/CacheHelper.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CacheHelper.swift
+//  camPod
+//
+//  Created by Janco Erasmus on 2020/04/20.
+//
+
+import Foundation
+
+public class CacheHelper {
+
+    public init() { }
+
+    func saveImage(imageName: String, image: UIImage) {
+        guard let documentsDirectory = FileManager.default.urls(for: .documentDirectory,
+                                                                in: .userDomainMask).first else { return }
+
+        let fileName = imageName
+        let fileURL = documentsDirectory.appendingPathComponent(fileName)
+
+        guard let data = image.jpegData(compressionQuality: 1) else { return }
+
+        if (FileManager.default.fileExists(atPath: fileURL.path)) {
```

Comment:
Very minor but there's no need for extra brackets here. You can just have:
`FileManager.default.fileExists(atPath: fileURL.path)`

Reply:
Cool thanks, Codacy also told me hahaha